source/_integrations/ibeacon.markdown

```diff
@@ -41,6 +41,10 @@ Consider setting up your iBeacons with a schema similar to the following:
 
 iBeacon devices that do not have stable Major and Minor values are not supported. The system automatically removes iBeacon devices with unstable Major and Minor values once ten (10) or more Major and Minor values have been seen with the same UUID from an iBeacon device with a fixed MAC address.
 
+## Considering an iBeacon Away
+
+Due to various factors such as individual system settings and iBeacon firmware, iBeacons may not be marked as "Away" immediately. This could take several minutes.
```

Comment:
I think we can definitely say that will not be

Reply:
Updated accordingly.